### PR TITLE
Speed up file listing rendering.

### PR DESCRIPTION
**Pull request by shylux**
The html2latex conversion was too slow for workspaces with a huge amount of files.
This commit skips the html step and generates latex directly from template.

Reproduce the problem by loading ~300 files in a workspace and generate the pdf. This will take 30+ seconds.
This PR reduces the time to create the pdf to almost 0.

### CODE REVIEW

File: ftw/workspace/latex/templates/files_listing.pt [hunk 1]
```diff
@@ -5,42 +5,27 @@
...
-                <td tal:content="item/effective"
-                    class="border-bottom" />
-                <td tal:content="item/modified"
-                    class="border-bottom" />
-                <td tal:content="item/creator"
-                    class="border-bottom" />
-            </tr>
-        </tbody>
-    </table>
+    \multicolumn{1}{p{0.5\tablewidth}}{\textbf{\textbf{<tal:omit tal:omit-tag="" i18n:translate="latex_files_title">Title</tal:omit>}}} &
```
Comment (jone) [inline ftw/workspace/latex/templates/files_listing.pt]: The title is not converted to LaTeX. 
For example using umlauts or special characters will break here.

Be aware that within Plone `pdflatex` is run in nonstop-mode, so you need to download the ZIP-file and run it with `pdflatex` manually in order to detect problems.

File: ftw/workspace/latex/templates/files_listing.pt [hunk 2]
```diff
@@ -5,42 +5,27 @@
     tal:omit-tag="python: True"
```
Comment (jone) [inline ftw/workspace/latex/templates/files_listing.pt]: Usually we use Mako-Templates for LaTeX, not page templates. Is there a reason for still using page template other than convenience?

Example: https://github.com/4teamwork/ftw.meeting/blob/master/ftw/meeting/latex/templates/tasklisting.tex
(But do not use `supertabular` 😜 )

**Comment by shylux** [comment]
@jone updated. Still generates almost instantly.

**Comment by jone** [comment]
Awesome, thanks!

**Pull request merged**